### ISSUE TITLE
bug(linter): false positive in oxc/const-comparisons

### ISSUE THREAD

**Issue by camc314** [oxc-project/oxc#8163]
https://github.com/oven-sh/bun/blob/c2a77cf7ec9de9eadf938046bdf78e58561c8a6d/src/js/builtins/EventSource.ts#L322

```
  × oxc(const-comparisons): This comparison will always evaluate to true
     ╭─[src/js/builtins/EventSource.ts:322:19]
 321 │               self.#received_length += chunks.length;
 322 │               if (self.#received_length >= self.#content_length) {
     ·                   ─────────────────────────────────────────────
 323 │                 self.#state = 2;
     ╰────
  help: Because `self.#received_length` will always be equal to itself
```
